packages/features/additionalTimezones/additionalTimezones.json

```diff
@@ -1,64 +1,64 @@
 {
   "timezones": [
     {
-      "name": "Eastern Time - US & Canada",
-      "identifier": "America/New_York"
+      "timezone": "Eastern Time - US & Canada",
+      "city": "America/New_York"
     },
     {
-      "name": "Pacific Time - US & Canada",
-      "identifier": "America/Los_Angeles"
+      "timezone": "Pacific Time - US & Canada",
+      "city": "America/Los_Angeles"
     },
     {
-      "name": "Central Time - US & Canada",
-      "identifier": "America/Chicago"
+      "timezone": "Central Time - US & Canada",
+      "city": "America/Chicago"
     },
     {
-      "name": "Mountain Time - US & Canada",
-      "identifier": "America/Denver"
+      "timezone": "Mountain Time - US & Canada",
+      "city": "America/Denver"
     },
     {
-      "name": "Atlantic Time - Canada",
-      "identifier": "America/Halifax"
+      "timezone": "Atlantic Time - Canada",
+      "city": "America/Halifax"
     },
     {
-      "name": "Eastern European Time",
-      "identifier": "Europe/Bucharest"
+      "timezone": "Eastern European Time",
+      "city": "Europe/Bucharest"
     },
     {
-      "name": "Central European Time",
```

Comment:
The updated naming doesn't make a lot of sense

Reply:
I know, I did it just to keep the naming convections of the code streamlined, as this convections are used in timezone.ts by the developers/contributers  of cal.com.

Reply:
Yes - but it looks like these are the wrong way around, in the cityTimezones logic the city would be "Eastern European Time" and the "timezone": Europe/Bucharest

Reply:
ok, give me a min, I'll fix that.

Reply:
I'm refactoring the 'city' to label for improved readability.

Reply:
ok, no issues.